app/views/steps/trader_scheme/show.html.erb

```diff
@@ -7,6 +7,8 @@
 
   <%= f.govuk_collection_radio_buttons :trader_scheme, @step.options, :id, :name, legend: { text: "#{trader_scheme_header}?", size: 'xl', tag: 'h1' }, hint: { text: "#{trader_scheme_body}" }, include_hidden: true %>
 
+  <p><%= trader_scheme_body_extra_text %></p>
```

Comment:
This way, we might have an empty paragraph. Should we put <p> paragraph inside the  trader_scheme_body_extra_text method.

Reply:
Yeah true i did think about this